src/WebJobs.Script/Rpc/Configuration/SupportedWorkerPathOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc.Configuration
+{
+    public static class SupportedWorkerPathOptions
+    {
+        static SupportedWorkerPathOptions()
+        {
+            Runtime = new Dictionary<string, WorkerPathOptions>();
+
+            Runtime[LanguageWorkerConstants.PythonLanguageWorkerName] = new WorkerPathOptions()
+            {
```

Comment:
Adding language specific versions in the host does not work. This information needs to be coming from the worker.config.json. Please add SupportedArchitectures, SupportedPlatforms and SupportedVersions to the config file

Reply:
Just noting that this would require the worker config to always contain the supported features if the user opts into them.